booth.spec

```diff
@@ -1,5 +1,11 @@
 %bcond_with html_man
 
+%if 0%{?fedora} > 18 || 0%{?centos} > 6 || 0%{?rhel} > 6
```

Comment:
Shouldn't this be the other way around? I currently get:

 [   59s] configure: error: logging provider required (libplumb, or libqb when --without-glue)